x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/components/installed_integration_action_menu.tsx

```diff
@@ -56,6 +57,22 @@ export const InstalledIntegrationsActionMenu: React.FunctionComponent<{
     );
   }, [selectedItems, startServices, bulkUpgradeIntegrations]);
 
+  const openUninstallModal = useCallback(() => {
+    setIsOpen(false);
+    const ref = startServices.overlays.openModal(
+      toMountPoint(
+        <ConfirmBulkUninstallModal
+          onClose={() => {
+            ref.close();
+          }}
+          onConfirm={() => bulkUninstallIntegrations(selectedItems)}
```

Comment:
Uninstall won't work if there are policies, should we offer a checkbox to force uninstall?

Reply:
I am not sure we should offer a force option, user will get the error results, and it seems to me it's probably better if they fix the issue by either removing agent or related package policy. And keep the force only in the API for support, wdyt?